engine/src/test/java/io/camunda/zeebe/engine/processing/deployment/MultiPartitionDeploymentLifecycleTest.java

```diff
@@ -218,6 +219,7 @@ public void shouldRejectCompleteDeploymentDistributionWhenAlreadyCompleted() {
             RecordType.COMMAND, RecordType.COMMAND, RecordType.COMMAND_REJECTION);
   }
 
+  @Ignore("Regression https://github.com/camunda/zeebe/issues/13233")
```

Comment:
❓ I am so confused why we backport this if the problem doesn't exist on this version?